omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/ConceptController1_8Test.java

```diff
@@ -291,6 +285,16 @@ public void shouldNotReturnAnythingWhenConceptDoesntMatchClass() throws Exceptio
 		hits = result.get("results");
 		
 		assertThat(hits, is(empty()));
+
+		//Should not find it when it exactly matches
+		name = "CD4 COUNT";
```

Comment:
Should we make the variable to be `final`?

Reply:
No I don’t believe we should since the variable is used earlier in the test with a different value. I could split this addition into a different test but this pattern is used throughout the test file for testing searchType parameter.